main.go

```diff
@@ -2,12 +2,21 @@ package main
 
 import (
 	"context"
+	"os"
 
 	"github.com/spf13/cobra"
 
 	"github.com/ollama/ollama/cmd"
+	"github.com/ollama/ollama/llama/runner"
 )
 
 func main() {
+	if len(os.Args) >= 2 {
+		if os.Args[1] == "_runner" {
```

Comment:
I'd just make this a hidden `runner` command here: https://github.com/ollama/ollama/blob/main/cmd/cmd.go#L1443